### PR TITLE
BENTO-2191: OverlayWindow component

### PR DESCRIPTION
**Pull request by JoonLeeNIH**
Is `master` the correct branch for this?

### CODE REVIEW

**Comment by iksheth** [comment]
> Is `master` the correct branch for this?

@JoonLeeNIH Yes. Master is correct branch for PR.

File: README.md [hunk 1]
```diff
@@ -46,6 +54,40 @@ const options = {
 };
 <MUIDatatable columns={columns} data={data} options={options} selectCellPostion="right" />
 ```
+
```
Comment (iksheth) [inline README.md:57]: @JoonLeeNIH Can we create a separate readme under "src/components/OverlayWindow/"? You can name it "README.md"

File: src/components/OverlayWindow/OverlayWindowView.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+/**
+ * TODO - Allow styles to be passed (eg: `classes` prop and className)
```
Comment (iksheth) [inline src/components/OverlayWindow/OverlayWindowView.js:2]: @JoonLeeNIH Is it possible to pass Styles as a prop currently?

**Review by iksheth — CHANGES_REQUESTED** [review]
@JoonLeeNIH There is one change requested and One question for clarification.

File: src/components/OverlayWindow/OverlayWindowView.js (hunk 2 above)
Comment (JoonLeeNIH) [reply src/components/OverlayWindow/OverlayWindowView.js:2]: No

File: README.md (hunk 1 above)
Comment (JoonLeeNIH) [reply README.md:57]: Sure. I have now added a README for this specific component

**Pull request merged**